apps/concierge_site/lib/helpers/session_helper.ex

```diff
@@ -11,9 +11,9 @@ defmodule ConciergeSite.SessionHelper do
 
   @doc "Signs in a user with Guardian and redirects to the appropriate route."
   @spec sign_in(Conn.t(), User.t()) :: Conn.t()
-  def sign_in(conn, user, claims \\ %{}) do
+  def sign_in(conn, user) do
     conn
-    |> ConciergeSite.Guardian.Plug.sign_in(user, claims)
+    |> ConciergeSite.Guardian.Plug.sign_in(user)
```

Comment:
The library already defaults to `%{}` for the `claims` argument so we don't need to supply it.